packages/file_selector/file_selector_linux/lib/src/messages.g.dart

```diff
@@ -0,0 +1,195 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+// Autogenerated from Pigeon (v22.4.1), do not edit directly.
+// See also: https://pub.dev/packages/pigeon
+// ignore_for_file: public_member_api_docs, non_constant_identifier_names, avoid_as, unused_import, unnecessary_parenthesis, prefer_null_aware_operators, omit_local_variable_types, unused_shown_name, unnecessary_import, no_leading_underscores_for_local_identifiers
+
+import 'dart:async';
+import 'dart:typed_data' show Float64List, Int32List, Int64List, Uint8List;
+
+import 'package:flutter/foundation.dart' show ReadBuffer, WriteBuffer;
+import 'package:flutter/services.dart';
+
+PlatformException _createConnectionError(String channelName) {
+  return PlatformException(
+    code: 'channel-error',
+    message: 'Unable to establish connection on channel: "$channelName".',
+  );
+}
+
+/// A Pigeon representation of the GTK_FILE_CHOOSER_ACTION_* options.
+enum PlatformFileChooserActionType {
+  open,
+  chooseDirectory,
+  save,
+}
+
+/// A Pigeon representation of the Linux portion of an `XTypeGroup`.
+class PlatformTypeGroup {
+  PlatformTypeGroup({
+    this.label = '',
+    this.extensions = const <String>[],
+    this.mimeTypes = const <String>[],
+  });
+
+  String label;
+
+  List<String> extensions;
+
+  List<String> mimeTypes;
+
+  Object encode() {
+    return <Object?>[
+      label,
+      extensions,
+      mimeTypes,
+    ];
+  }
+
+  static PlatformTypeGroup decode(Object result) {
+    result as List<Object?>;
+    return PlatformTypeGroup(
+      label: result[0]! as String,
+      extensions: (result[1] as List<Object?>?)!.cast<String>(),
+      mimeTypes: (result[2] as List<Object?>?)!.cast<String>(),
+    );
+  }
+}
+
+/// Options for GKT file chooser.
```

Comment:
GKT > GTK :)

Reply:
Doh! I'll try to remember to fix that next time we touch the file.